doc/source/user_guide/reshaping.rst

```diff
@@ -296,6 +296,25 @@ For instance,
    cheese.melt(id_vars=['first', 'last'])
    cheese.melt(id_vars=['first', 'last'], var_name='quantity')
 
+When transforming a DataFrame using :func:`~pandas.melt`, original the index values can be kept around by setting the ``ignore_index`` parameter to ``False``. This will however duplicate the index values.
+
+.. ipython:: python
+
+   index = pd.MultiIndex.from_tuples([('person', 'first'), ('person', 'second')])
+   cheese = pd.DataFrame({'first': ['John', 'Mary'],
+                          'last': ['Doe', 'Bo'],
+                          'height': [5.5, 6.0],
+                          'weight': [130, 150]},
+                         index=index)
+   cheese
+
+.. ipython:: python
```

Comment:
no need to repeat the directive here, as it creates an extra box and splits the example